jerry-core/ecma/builtin-objects/ecma-builtins.c

```diff
@@ -331,6 +402,24 @@ ecma_instantiate_builtin (ecma_builtin_id_t id) /**< built-in id */
 {
   switch (id)
   {
+#define BUILTIN(a, b, c, d, e)
+#define BUILTIN_CALLABLE(builtin_id, \
```

Comment:
Can't we just call the other define in the BUILTIN_CALLABLE here?
ex.:
```
#define BUILTIN_CALLABLE(a) BUILTIN(a)
```

Reply:
Good idea